README.md

```diff
@@ -94,3 +96,67 @@ router.route('/users/:userId')
 router.param('userId', users.findById.bind(users));
 module.exports = router;
 ```
+
+### use aggregation pipeline
+
+The `index` method of `BaseController` now supports MongoDB aggregation pipeline. To make
+use of it define an `aggregationPipeline(query)` function in the corresponding model and return
+the desired aggregation pipeline.
+
+Example excerpt from Payment model:
+
+```javascript
+const lookups = [{
+  from: 'subjects',
+  localField: 'subject',
+  foreignField: '_id',
+  as: 'subject'
+}, {
+  from: 'accesscodes',
+  localField: 'accessCode',
+  foreignField: '_id',
+  as: 'accessCode'
+}, {
+  from: 'treatments',
+  localField: 'treatment',
+  foreignField: '_id',
+  as: 'treatment'
+}];
+
+PaymentSchema.statics.aggregationPipeline = query => {
```

Comment:
Is the naming `aggregationPipeline` correct? What does it do? It parses the query to get a pipeline for aggregation.

Reply:
or maybe just `aggregatePipeline`